src/addons/transitionDimensions.ts

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable */
```

Comment:
Yeah, this file is not being used and was written before I worked on the better FLIP layout technique, so it's not going to be exported or anything. Doesn't serve a purpose for now.

Reply:
@atomiks https://twitter.com/markdalgleish/status/1118286402280546304 :smile:

Reply:
Fair but also https://twitter.com/LashaKrikheli/status/1118493450847031297

Since the FLIP technique does not have its own wrapper yet, I'm keeping it fresh in my mind on master to refer to it quickly. I'll delete it once there's an API for the FLIP technique 😛